### PR TITLE
Taggle style improvements

### PR DESCRIPTION
**Pull request by thinkh**
Related PRs:

https://github.com/sgratzl/taggle2/pull/93
https://github.com/Caleydo/lineupjs/pull/312

### CODE REVIEW

File: src/styles/_vars.scss [hunk 1]
```diff
@@ -3,4 +3,5 @@ $engine_assets: "./assets" !default;
 $engine_even_color: #f7f7f7 !default;
 $engine_loading_image: url('#{$engine_assets}/loading.svg') !default;
 $engine_loading_static_image: url('#{$engine_assets}/loading_s.svg') !default;
-$engine_hover_outline_color: 2px solid orange !default;
+$engine_hover_outline_width: 2px;
```
Comment (sgratzl) [inline src/styles/_vars.scss]: should have the `!default` flag at the end

File: src/styles/_layout.scss [hunk 2]
```diff
@@ -60,6 +60,9 @@
   }
 
   > main:not(.scrolling) > article > div {
+    /* fix: dirty overlapping of orange outline with neighboring rows */
+    margin: $engine_hover_outline_width 0;
```
Comment (sgratzl) [inline src/styles/_layout.scss]: as discussed before, in the end we want to switch to a border (transparent vs color) approach for highlighting. requires #4 

I will merge it for now but we have to keep that in mind

File: src/styles/_vars.scss (hunk 1 above)
Comment (thinkh) [reply src/styles/_vars.scss]: I simply forgot that. -> Fixed.

File: src/styles/_layout.scss (hunk 2 above)
Comment (thinkh) [reply src/styles/_layout.scss]: Yes, we can try this in the next iteration.

**Pull request merged**